### DISCUSSION TITLE
k0s script

### DISCUSSION THREAD

**Discussion by trendstyle** [tteck/Proxmox#1980] [Help Needed]
it seems this script is not functioning as intended anymore.

kubernates on lxc is a real pain in the A$%, i thought this one would be awesome,
but unfortunnately, the pods are not functioning correctly and are stuck pending, when runnning
k0s kubectl get pods --all-namespaces

no matter how hard i try i cannot get anything to function

**Comment by tteck** [comment]
Removed
